sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-07-16)
 
+### Features Added
+- Added support for [RequestOptions](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md#using-requestoptions-to-customize-behavior) on protocol methods.
+- Added support for API management.
+- Simplified dependencies.
+
+### Bugs Fixed
```

Comment:
Hi @KrzysztofCwalina, is https://github.com/Azure/azure-webpubsub/issues/47 fixed in this release?

Reply:
The new swagger has minLength 1 paths, so it's theoretically fixed. It would be good to validate the fix too, but the repro steps have a broken link to repro code.